### PR TITLE
Update terra-jupyter-aou to Hail 0.2.98

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

**Comment by danking** [comment]
@calbach LMK if you prefer someone else to be assigned!

**Comment by Qi77Qi** [comment]
cc @yonghaoy ...btw, we recently update gatk image as well, and that change hasn't made it to AoU yet

**Comment by calbach** [comment]
@danking I'm no longer working on AoU, I defer to @yonghaoy for identifying the right reviewer from the AoU team going forward.

File: config/conf.json [hunk 1]
```diff
@@ -83,9 +83,7 @@
             "tools" : [
                 "python"
             ],
-            "packages" : {
-                
-            },
```
Comment (yonghaoy) [inline config/conf.json:88]: revert those untouched change

**Review by yonghaoy — CHANGES_REQUESTED** [review]
https://github.com/DataBiosphere/terra-docker/pull/347/files 2.1.9 is a bad version we should skip that. 
We should use 2.1.10 instead. 
Example: https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-aou/CHANGELOG.md#214---2022-06-03t183522582705z

**Comment by Qi77Qi** [comment]
superceded by https://github.com/DataBiosphere/terra-docker/pull/351

**Pull request closed without merging**